### ISSUE TITLE
gau: command not found

### ISSUE THREAD

**Issue by hassankhan98** [lc/gau#11]
GO111MODULE=on go get -u -v github.com/lc/gau after this when i try to run 
gau: command not found?

**Comment by lc** [comment]
Any more details, like your `go version` and $GOPATH?

Also can you show the output of `GO111MODULE=on go get -u -v github.com/lc/gau`? Mine works fine:

```
$ GO111MODULE=on go get -u -v github.com/lc/gau

go: downloading github.com/lc/gau v1.0.2
go: github.com/lc/gau upgrade => v1.0.2
github.com/lc/gau/providers
github.com/lc/gau
```

**Comment by hassankhan98** [comment]
go version go1.13.3 darwin/amd6

![Screen Shot 2020-05-17 at 1 25 57 PM](https://user-images.githubusercontent.com/33081303/82159281-fe99a600-9841-11ea-8b64-510bb49a9af0.png)

**Comment by lc** [comment]
Do you have your $GOPATH variable set? `echo $GOPATH`

**Comment by hassankhan98** [comment]
No Sir How can i do that?

**Comment by lc** [comment]
Ok, once you have that set this should work! Here's how to do it: https://www.digitalocean.com/community/tutorials/how-to-install-go-and-set-up-a-local-programming-environment-on-ubuntu-18-04

tl;dr:

```
mkdir -p $HOME/go/{bin,pkg,src}
```

Then edit your `~/.bashrc` (or `~/.zshrc` if you're using zsh) and add the following to the file:

```
export GOPATH=$HOME/go
export PATH=$PATH:$GOPATH/bin
```
